src/App.css

```diff
@@ -93,11 +225,49 @@ hr {
   gap: 30px;
 }
 
+.f-head {
+  font-family: "Montserrat", sans-serif;
+  color: #888;
+  letter-spacing: -0.18px;
+  font-weight: 700;
+}
+
 .book-name,
 .book-title,
 #category-option {
-  width: 30%;
+  width: 25%;
   height: 60px;
-  font-size: 25px;
-  text-align: center;
+  font-size: 20px;
+  border: 1px solid #ccc;
+  font-family: "Montserrat", sans-serif;
+  border-radius: 5px;
+  padding-left: 50px;
+}
+
+.submit-btn {
+  font-size: 20px;
+  width: 20%;
+  border-radius: 5px;
+  background-color: #0290ff;
+  letter-spacing: 0.5px;
+  font-weight: 700;
+  font-family: "Roboto Slab", serif;
+  color: #fff;
+  border-style: none;
+  cursor: pointer;
+}
+
+#category-option {
+  background-color: #fff;
+}
+
+/* category page */
+.cat-page-cont {
+  display: flex;
+  height: 400px;
+  background-color: #fafafa;
+  font-size: 50px;
+  align-items: center;
+  padding-left: 170px;
+  font-family: "Montserrat", sans-serif;
 }
```

Comment:
- Please ensure your project matches the Zeplin design as much as possible. You are almost there 😉
Your Design | Zeplin Design
![Screenshot from 2023-01-28 01-23-13](https://user-images.githubusercontent.com/44624138/215229858-41bcf023-59a3-4810-99d7-9edc98a70b2f.png) | ![bookstore](https://user-images.githubusercontent.com/44624138/215229892-35133301-0a47-4f61-9437-c65cb58b1bc4.png)